src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,144 @@
+package core.basesyntax.service;
+
+import static core.basesyntax.service.RegistrationServiceImpl.AGE;
+import static core.basesyntax.service.RegistrationServiceImpl.EXIST_MESSAGE;
+import static core.basesyntax.service.RegistrationServiceImpl.LESS_MESSAGE;
+import static core.basesyntax.service.RegistrationServiceImpl.LOGIN;
+import static core.basesyntax.service.RegistrationServiceImpl.LOGIN_PASSWORD_MIN_LENGTH;
+import static core.basesyntax.service.RegistrationServiceImpl.MIN_AGE;
+import static core.basesyntax.service.RegistrationServiceImpl.NULL_MESSAGE;
+import static core.basesyntax.service.RegistrationServiceImpl.PASSWORD;
+import static core.basesyntax.service.RegistrationServiceImpl.RegistrationException;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String LOGIN_OK = "userLogin";
+    private static final String LOGIN_NOT_OK = "login";
+    private static final String PASSWORD_OK = "userPassword";
+    private static final String PASSWORD_NOT_OK = "pass";
+    private static final int AGE_OK = 18;
+    private static final int AGE_NOT_OK = 17;
+    private static final int AGE_ZERO = 0;
+    private static final String EMPTY_STRING = "pass";
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User(LOGIN_OK, PASSWORD_OK, AGE_OK);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void registerProperUserOk() {
```

Comment:
as it is stated in the common mistakes, same in other test methods
```suggestion
    void register_ProperUser_Ok() {
```